kava/operations_test.go

```diff
@@ -573,41 +578,339 @@ func TestFeeToOperations(t *testing.T) {
+			status: SuccessStatus,
+		},
+		{
+			name:   "hard.MsgBorrow",
+			logs:   readABCILogFromFile("hard-borrow-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "hard.MsgRepay",
+			logs:   readABCILogFromFile("hard-repay-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "hard.MsgLiquidate",
+			logs:   readABCILogFromFile("hard-liquidate-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "auction.MsgPlaceBid",
+			logs:   readABCILogFromFile("auction-bid-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "bep3.MsgCreateAtomicSwap",
+			logs:   readABCILogFromFile("bep3-create-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "bep3.MsgRefundAtomicSwap",
+			logs:   readABCILogFromFile("bep3-refund-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "bep3.MsgClaimAtomicSwap",
+			logs:   readABCILogFromFile("bep3-claim-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "cdp.MsgCreateCDP",
+			logs:   readABCILogFromFile("cdp-create-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "cdp.MsgDeposit",
+			logs:   readABCILogFromFile("cdp-deposit-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "cdp.MsgWithdraw",
+			logs:   readABCILogFromFile("cdp-withdraw-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "cdp.MsgDrawDebt",
+			logs:   readABCILogFromFile("cdp-draw-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "cdp.MsgRepayDebt",
+			logs:   readABCILogFromFile("cdp-repay-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "cdp.MsgLiquidate",
+			logs:   readABCILogFromFile("cdp-liquidate-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "kava.SubmitProposal",
+			logs:   readABCILogFromFile("committee-submit-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "kava.MsgVote",
+			logs:   readABCILogFromFile("committee-vote-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "incentive.MsgClaimUSDXMintingReward",
+			logs:   readABCILogFromFile("incentive-claim-usdx-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "incentive.MsgClaimHardReward",
+			logs:   readABCILogFromFile("incentive-claim-hard-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "pricefeed.MsgPostPrice",
+			logs:   readABCILogFromFile("pricefeed-post-tx-response.json"),
+			status: SuccessStatus,
+		},
+		{
+			name:   "cosmos-sdk.MsgSend",
+			logs:   readABCILogFromFile("msg-send-tx-response.json"),
+			status: SuccessStatus,
 		},
 	}
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			runAndAssertOperationInvariants(t, tc.opType, func(otc *operationTestCase) []*types.Operation {
-				msg, log := tc.createFn(otc.coins)
-				ops := MsgToOperations(msg, log, &otc.status, otc.index)
+			for _, log := range tc.logs {
```

Comment:
With below change of ensuring fixure as a single log, will be easy to bring back the operation test case wrapper.